cli/src/cli.rs

```diff
@@ -206,6 +207,22 @@ pub enum CliCommand {
     },
     TimeElapsed(Pubkey, Pubkey, DateTime<Utc>), // TimeElapsed(to, process_id, timestamp)
     Witness(Pubkey, Pubkey),                    // Witness(to, process_id)
+    // Nonce commands
+    CreateNonceAccount {
+        nonce_account: KeypairEq,
+        lamports: u64,
+    },
+    NewNonce(KeypairEq),
+    GetNonce(Pubkey),
```

Comment:
Some A-Z requests... 
(1) Could we alphabetize the Nonce commands? I know it makes sense to put them in usage order, and we used to do a lot of that here, but it made it pretty difficult to find anything in this ginormous file.
(2) Could you move the "Nonce commands" block above "Program Deployment"? (and same request for the `parse_` match block and the `process_` match block)